src/SourceLink.GitWeb/GetSourceLinkUrl.cs

```diff
@@ -0,0 +1,35 @@
+﻿// Copyright (c) Microsoft. All Rights Reserved. Licensed under the Apache License, Version 2.0. See
+// License.txt in the project root for license information.
+
+using System;
+using Microsoft.Build.Framework;
+using Microsoft.Build.Tasks.SourceControl;
+
+namespace Microsoft.SourceLink.GitWeb
+{
+    /// <summary>
+    /// The task calculates SourceLink URL for a given SourceRoot. If the SourceRoot is associated
+    /// with a git repository with a recognized domain the <see cref="SourceLinkUrl"/> output
+    /// property is set to the content URL corresponding to the domain, otherwise it is set to
+    /// string "N/A".
+    /// </summary>
+    public sealed class GetSourceLinkUrl : GetSourceLinkUrlGitTask
+    {
+        protected override string HostsItemGroupName => "SourceLinkGitWebHost";
+        protected override string ProviderDisplayName => "GitWeb";
+
+        protected override string BuildSourceLinkUrl(Uri contentUri, Uri gitUri, string relativeUrl, string revisionId, ITaskItem hostItem)
+        {
+            var trimLeadingSlash = relativeUrl.TrimStart('/', '\\');
```

Comment:
>'\\' [](start = 62, length = 4)

It shouldn't be necessary to trim backslash.

Reply:
Done